x-pack/plugins/observability_solution/synthetics/common/rules/synthetics_rule_field_map.ts

```diff
@@ -32,6 +32,10 @@ export const syntheticsRuleFieldMap: FieldMap = {
     type: 'text',
     required: false,
   },
+  'error.stack_trace': {
+    type: 'wildcard',
```

Comment:
How do you decide if a field is a wildcard or text? Is it only related to how the user searches a field?

Reply:
i followed it's type from ecs